sdk/iot/Azure.Iot.Hub.Service/tests/IotHubServiceTestEnvironment.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core.TestFramework;
+
+namespace Azure.Iot.Hub.Service.Tests
+{
+    // This class contains the configurations required to be set to run tests against the CI pipeline.
+    public class IotHubServiceTestEnvironment : TestEnvironment
+    {
+        public IotHubServiceTestEnvironment()
+            : base("iothub")
```

Comment:
This should be the constant defined in the TestSettings file. @azabbasi what should this value be for this client library?

Reply:
just `iot`

Reply:
`iot`

Reply:
q - why not "iothub"? is it supposed to be the service we are targeting, or is it an indicator of where the SDK sits in the repo?

Reply:
From what I know it is dictated by the folder we are in - Some kind of guidelines/rules from the Azure.Core team. @azabbasi knows.